addons/gforces/XEH_postInit.sqf

```diff
@@ -13,7 +13,7 @@ GVAR(playerIsVirtual) = false;
     ["unit", { // Add unit changed EH to check if player is either virtual (logic) or a UAV AI
         params ["_unit"];
         GVAR(playerIsVirtual) = ((getNumber (configOf _unit >> "isPlayableLogic")) == 1) ||
-        {(getText (configOf _unit >> "simulation")) == "UAVPilot"};
+        {unitIsUAV _unit};
         TRACE_3("unit changed",_unit,typeOf _unit,GVAR(playerIsVirtual));
```

Comment:
Switch the order of the arguments, so that with lazy eval it makes more sense.
Bring onto a single line.